textalyze.js

```diff
@@ -14,10 +6,17 @@
 function itemCounts(array) {
   let counts = new Map();
 
-  // Your code here.
-  // Run 'npm test' to see what tests need to pass.
+  for (const a of array) {
+    counts.has(a) ? counts.set(a, counts.get(a) + 1) : counts.set(a, 1);
+  }
 
   return counts;
 }
 
+const elements = ["a", "a", "a", "b", "b", "c"];
+console.log(`Count for [${elements}] is...`);
+itemCounts(elements).forEach((count, elem) => {
+  console.log(elem, count);
+});
```

Comment:
Check out [console.table](https://developer.mozilla.org/en-US/docs/Web/API/Console/table) — it's useful for quickly printing out tables in debugging situations.